doc/InfiniTimeVision.md

```diff
@@ -0,0 +1,32 @@
+# InfiniTime Vision
+
+The purpose of this document is to steer efforts towards a common goal, and as such should be taken into consideration with all developments.
+
+## InfiniTime
+
+InfiniTime is a community-built smartwatch firmware, providing freedom and privacy that's usually insufficient in wearable technology.
+
+InfiniTime is not to be used for medical or other health tracking purposes.
+
+## Core Principles
+
+- KISS (Keep It Short and Simple)
```

Comment:
Imho this is vague. What about something like `write simple code, expand later if needed`. Since it's a mono-codebase it's useless to build supposedly ‶future-proof″ abstractions right away. We can refactor code once we have a clearer view about what kind of abstraction would be a great fit.

Reply:
It's intentionally so. Also "Future-proof" abstractions are rarely short and simple.

Reply:
@Avamander this is literally what I said.